RELEASE_NOTES.md

```diff
@@ -6,6 +6,10 @@
 
 The Release Procedure is defined in the [CONTRIBUTING](CONTRIBUTING.md#release-procedure) document.
 
+## v6.5.0
+
+In this release we are adding validations to ibc-transfer msg receiver address and memo feilds. This release will resolve the latest incident of spam ibc-transfer transactions.
+
```

Comment:
let's be more descriptive here - we should let know to the users what's the limit of the receiver and memo field. Most of them will only read release notes.